controllers/components/ApiComponent.php

```diff
@@ -124,32 +124,35 @@ public function search($args)
         $resourceDao = MidasLoader::loadModel("Item")->initDao("Resource", $item->toArray(), "journal");
         $rating = MidasLoader::loadModel("Itemrating", 'ratings')->getAggregateInfo($item);
         $authors = join(", ", $resourceDao->getAuthorsFullNames());
+        // Check that the submission isn't under the "Private" folder of the given issue
+        $parentFolder = $item->getFolders();
+        if($parentFolder[0]->getName() != "Private") {
```

Comment:
@billhoffman Would something like this be more acceptable: https://github.com/josephsnyder/journal/commit/4f0713797e504824816d9b384c560cef4b0adf12#diff-51dd940ce35133e0e62ca93e5aa6e08cR127